### ISSUE TITLE
Refactoring Frontend

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#85]
## User story
1. As a develeper
2. I need well structured code
3. So that I can implement new features more easily

## Acceptance criteria
* [ ] Make code more readable and structured

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
